src/govuk/components/radios/index.js

```diff
@@ -7,7 +7,7 @@ function Radios(props) {
   // Map React-like `value` top level prop to the child items' checked status
   const processedItems = items.map(item => ({
     ...item,
-    ...(value && { checked: item.value === value })
+    checked: item.value === value
```

Comment:
Before this change, React was complaining about the input changing from uncontrolled to controlled. This resolves that